src/teleop/gui/src/components/CameraSelection.vue

```diff
@@ -1,14 +1,15 @@
 <template>
   <div class="wrap">
-    <div v-for="j in 3" class="buttons">
+    <div v-for="j in 3" :key="j" class="buttons">
```

Comment:
Out of curiosity what's the reasoning behind adding the key here? Doesn't this not change anything for something so simple?